src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/mapper/induction/InductionEntityListManager.kt

```diff
@@ -0,0 +1,58 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.database.jpa.mapper.induction
+
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.app.database.jpa.entity.induction.EntityKeyAware
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.induction.DomainKeyAware
+
+/**
+ * Provides a centralised place to update, add or remove existing JPA entities (and thereby apply changes to the
+ * database). Manages changes between lists of domain model objects (provided via the REST API) and the equivalent
```

Comment:
Minor, but I'm not sure we should or want to mention REST API here do we?
This is a helper/manager class for helping map between domain and the corresponding entity objects. I'm not sure the REST API or it's request/response classes are involved here are they?

Reply:
Haha, I thought you might not like that! For me, the importance of the separation of layers is for flexibility and maintenance of the code itself (e.g. in the unlikely scenario that we have a new API technology in the future). It's not so much to abstract it to the point that we almost pretend we don't know what the API technology is here. 

I'm less bothered about a comment to be honest, since it won't break anything if the end client is changed, but I appreciate we need to keep comments up-to-date. Ultimately we all know the API is called via REST and this explanation is trying to provide some context, but I could change it to something like the "end client API" or if you prefer? :)

Reply:
Changed `REST` to `external` 😄